.forge-snapshots/swap with hooks.snap

```diff
@@ -1 +1 @@
-123297
\ No newline at end of file
+142010
```

Comment:
guessing this is because our all hooks address has less zeros now?

Reply:
no its because this gas test for "swap with hooks" wasnt calling any hooks 🙃

Reply:
aha and now it is yes?